runtime/jilgen/module.xml

```diff
@@ -98,6 +101,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<command line="cd $(UMA_PATH_TO_ROOT) &amp;&amp; qemu-arm -r '9' ./constgen" type="all">
 				<include-if condition="spec.linux_arm.*"/>
 			</command>
+			<command line="cd $(UMA_PATH_TO_ROOT)" type="all">
+				<include-if condition="spec.linux_ztpf.*"/>
+			</command>
+			<command line="if [ -x ../build_constgen.sh ] &amp;&amp; [ ! -e ../constgen.completed ] ; then ../build_constgen.sh ; fi" type="all">
```

Comment:
Better to combine these two commands into a single one to ensure they run as group.

Reply:
Done.